cli/deployment/deployment.go

```diff
@@ -0,0 +1,75 @@
+package deployment
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+)
+
+type Deployment struct {
+	Outputs    map[string]string
+	config     Config
+	configFile string
+}
+
+func (d Deployment) ConfigFile() string {
+	return d.configFile
+}
+
+func NewDeployment(config Config, outputs map[string]map[string]string) *Deployment {
+	file, err := ioutil.TempFile("", config.Name)
+	defer file.Close()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Here you only log the error. In some cases this error may be rather similar to an error from `file.Write()`, so adding some disambiguating context around the log call will be helpful to differentiate the two.